### ISSUE TITLE
Crash on loading Native\AssetPackages

### ISSUE THREAD

**Issue by OrderWOPower** [szszss/TpacTool#7]
Crashes on loading Native\AssetPackages on Bannerlord 1.6.0.

**Comment by Areldir** [comment]
+1 

Would greatly appreciate an update to 1.6.0

**Comment by hunharibo** [comment]
Same, i am afraid to try and debug this myself :(

**Comment by Areldir** [comment]
Doesn't look like there's been any movement here. @szszss is this program still being supported?

**Comment by hunharibo** [comment]
I ended up forking it and hacking it to work: https://github.com/hunharibo/TpacTool/releases/tag/0.3.2.1

**Comment by markaltmaier** [comment]
don't follow @hunharibo repo it's full of malicious malwares

**Comment by Areldir** [comment]
> don't follow @hunharibo repo it's full of malicious malwares

Evidence? I and several modders I know use it without any issues, and have never reported any issues with malware being detected.

**Comment by markaltmaier** [comment]
@Areldir just download antivirus like Norton or MacAfee and you'll see the result
The original repo has no issues except version compatibility, but @hunharibo repo it does contain malicious malwares

**Comment by hunharibo** [comment]
thats pretty rich considering I didn't add more dependencies or anything of the sort. Only ever changed cs files that were already present in the original repo. If you can't decide false positives for yourself, probably modding (or any advanced computer usage) is not for you.

**Comment by markaltmaier** [comment]
@hunharibo yes I totally agree, I should stick with the original repo as I just tried the lib on version 1.5.8 beta and it's working as intended. so I guess all I needed is to revert back to a previous version that the lib actually support, and the antivirus won't give me alerts anymore, so basically just stick with the original repo.
